### PR TITLE
Feature/119

### LINKED ISSUE #119: Add getPNRByTicketNumber method
This method will be a part of `AirService` and will use `TerminalService` in order to obtain PNR from terminal response.

PNR number is required to import PNR before calling getTicket if PNR was not imported and it's impossible to get extended ticketing information for ticket (See #114).

### PR DESCRIPTION
**Pull request by dchertousov**
Added getPNRByTicketNumber method.
Closes #119

### CODE REVIEW

File: src/Services/Air/Air.js [hunk 1]
```diff
@@ -81,6 +82,32 @@ module.exports = (settings) => {
...
+      const terminal = createTerminalService(settings);
+      const memo = {};
+      return terminal.executeCommand(`*TE/${options.ticketNumber}`)
+        .then(
+          (response) => {
+            memo.response = response;
+          }
+        )
+        .then(
+          () => terminal.closeSession()
```
Comment (shmuga) [inline src/Services/Air/Air.js]: Can be written shortly as `.then(terminal.closeSession)`.

File: src/Services/Air/Air.js [hunk 2]
```diff
@@ -81,6 +82,32 @@ module.exports = (settings) => {
...
+        )
+        .then(
+          () => {
+            try {
+              return memo.response.match(/RLOC [^\s]{2} ([^\s]{6})/)[1];
+            } catch (err) {
+              throw new AirRuntimeError.PnrParseError(memo.response);
```
Comment (shmuga) [inline src/Services/Air/Air.js]: Is there any reason for separate error type?

Comment (dchertousov) [inline src/Services/Air/Air.js]: In general it is for not throwing

TypeError: Cannot read property '1' of null

File: src/Services/Air/Air.js (hunk 1 above)
Comment (dchertousov) [inline src/Services/Air/Air.js]: Shortened

File: src/Services/Air/Air.js (hunk 2 above)
Comment (shmuga) [inline src/Services/Air/Air.js]: Ok. Let it be 👌

File: src/Services/Air/Air.js (hunk 1 above)
Comment (dchertousov) [inline src/Services/Air/Air.js]: @see #1e19df5

**Pull request merged**